docker/1.3.0/final/Dockerfile.cpu

```diff
@@ -0,0 +1,48 @@
+FROM ubuntu:16.04
+ARG framework_installable
+ARG py_version
+ARG framework_support_installable=sagemaker_mxnet_container-2.0.0.tar.gz
+
+# Validate that arguments are specified
+RUN test $framework_installable || exit 1 && \
+    test $py_version || exit 1
+
+WORKDIR /tmp
+
+RUN apt-get update && \
+    apt-get -y install libopencv-dev libopenblas-dev python python3 wget curl nginx
```

Comment:
always end a apt-get layer with \
 && rm -rf /var/lib/apt/lists/*

https://docs.docker.com/v17.09/engine/userguide/eng-image/dockerfile_best-practices/#run

Reply:
Install only one python version. You can use a variable for that https://github.com/aws/sagemaker-tensorflow-container/blob/script-mode/docker/1.10.0/Dockerfile.cpu#L45

Reply:
Add --no-install-recommends